evalml/pipelines/time_series_classifcation_pipelines.py

```diff
@@ -0,0 +1,204 @@
+import pandas as pd
+
+from evalml.objectives import get_objective
+from evalml.pipelines.classification_pipeline import ClassificationPipeline
+from evalml.problem_types import ProblemTypes
+from evalml.utils.gen_utils import (
+    _convert_to_woodwork_structure,
+    _convert_woodwork_types_wrapper,
+    _get_rows_without_nans,
+    drop_rows_with_nans,
+    pad_with_nans
+)
+
+
+class TimeSeriesClassificationPipeline(ClassificationPipeline):
+    """Pipeline base class for time series classifcation problems."""
+
+    def __init__(self, parameters, random_state=0):
+        """Machine learning pipeline for time series regression problems made out of transformers and a classifier.
```

Comment:
`classification` instead of `regression`

Reply:
Good catch!